### DISCUSSION TITLE
Create a Docker registry in hub.docker.com and/or add build/run instructions in the docs

### DISCUSSION THREAD

**Discussion by raulgd** [Kozea/Radicale#1596] [External Packaging]
Hi,

I'd like to use Radicale on my server, and all my services are dockerized.

I see on this code repo that there's a docker file, it would be nice if you guys created a free public docker hub account with the radicale image so it's very easy to get Radicale up and running.

Also, in the meantime, if someone could share detailed instructions on building and running Radicale Docker would be very nice, as I'm seeing the docker file and I don't know which is the volume where the data will be saved, so I can map that path to the docker host hard drive.

Thanks

**Comment by danielfdickinson** [comment]
I've created one based on the Dockerfile in Kozea/Radicale GitHub @ https://hub.docker.com/r/cshore/radicale/

**Comment by funkyfuture** [comment]
fyi, here's one for the Raspberry Pi that also allows setting configuration parameters per environment variables: https://hub.docker.com/r/funkyfuture/rpi-radicale/

for an official image there could/should be variants with dependencies for the different storage and authentication backends.

**Comment by amancevice** [comment]
Any update on this? It's pretty easy to set up an acct with Docker hub.

**Comment by bebehei** [comment]
So, there is already [kozea](https://hub.docker.com/u/kozea/) registered with an automated build. So I guess at least one member of the kozea organisation has to have access to this dockerhub account.

In combination with my PR, I think this should get finally solved.

**Comment by jasle** [comment]
I would be interested in a official docker image for radicale on the docker hub too.

Is there any help needed to get radicale on docker hub?

**Comment by Trolldemorted** [comment]
Can we get an official response on this? Are there official docker images available, if yes where? Will you push (security) updates in a timely manner?

If not, did you give up or are there problems we can help with? I see multiple open, but ignored docker-related PRs.

**Comment by dessalines** [comment]
This would be really nice to have.

**Comment by fajfer** [comment]
+1

**Comment by instantdreams** [comment]
Please consider creating an official docker image and hosting on ghcr.io and hub.docker.com or other locations.
